airflow/api/common/experimental/__init__.py

```diff
@@ -32,9 +31,9 @@ def check_and_get_dag(dag_id: str, task_id: Optional[str] = None) -> DagModel:
 
     dagbag = DagBag(
         dag_folder=dag_model.fileloc,
-        read_dags_from_db=conf.getboolean('core', 'store_serialized_dags')
+        read_dags_from_db=True
```

Comment:
```suggestion
```

Make it the default?

Reply:
I was a bit back and forth on that one too coz it has an expectation that the Serialized DAGs are already written by the Scheduler, so kept it as `False` by default.